lib/txgh/handlers/github_hook_handler.rb

```diff
@@ -156,6 +156,10 @@ def tx_resources_for(branch)
         end
       end
 
+      def tx_config
+        @tx_config = KeyManager.tx_config(project, repo, branch)
```

Comment:
Kind of tough to tell without context, but should the assignment operator be `||=` here?